src/AWS.Deploy.Orchestration/CdkAppSettingsSerializer.cs

```diff
@@ -23,8 +23,8 @@ public string Build(CloudApplication cloudApplication, Recommendation recommenda
                 projectPath,
                 recommendation.Recipe.Id,
                 recommendation.Recipe.Version,
-                session.AWSAccountId,
-                session.AWSRegion,
+                session.AWSAccountId ?? "",
+                session.AWSRegion ?? "",
```

Comment:
We should keep them nullable. Specially for region, logic can be different. We might want to fallback to default region depending on the requirement.

Reply:
Made them nullable.